src/Application.cpp

```diff
@@ -655,27 +655,34 @@ namespace weatherserver {
 
 int main(int argc, char* argv[]) {
 
-    //extracting current directory
-    std::string fullPath(argv[0]);
-    std::string execDir = fullPath.substr(0, fullPath.find_last_of("/\\") + 1); //including separator since we are adding file name later
-    std::cout << "Executable directory: " << execDir << std::endl;
+    std::string settingsPath = "";
+
+    if (argc > 1) {
+        settingsPath = argv[1];
+        std::cout << "Accepted custom settings file path as a parameter: " << settingsPath << std::endl;
+    }
+    else {
+        //default settings file location - current directory
+        settingsPath = "settings.json";
```

Comment:
Why executable directory path is no longer used by default?
Looks strange to me:
 - we copy `settings.json` file to this directory,
 - but try to load this file from _current directory_.

Just try to start the server from project root directory like this: `build/bin/Release/aqw-opcua-server`.